README.md

```diff
@@ -74,45 +103,53 @@ Total Flash memory (text + data + misc): 60554 bytes
 Image: .\BUILD\<TARGET_NAME>\ARM\mbed-os-example-bootloader.bin
 ```
 
-It creates two binary files. The original uncombined image is in the output directory, <project-name>_application.bin, and the bootloader image is <project-name>.bin.
+After the build completes, the build system saves two binary images in the `BUILD/<TARGET_NAME>/<TOOLCHAIN_NAME>` directory. The original unpadded image is saved with the name `<project-name>_application.bin`. The padded bootloader image (used for combining with applications) is saved with the name `<project-name>.bin`.
```

Comment:
Could you use the word "create" instead of "save" in both places? better yet, I recall rewording the second sentence to say something like "The unpadded image is named `project-blah-blah`."